conflowio/conflow
# openapi work

- Better parameter binding + testing
- Implement BindBody, rework validatevalue and pointer vs value handling
- Sort out schema reference resolution
- Validation improvements
- Allow to expose node to conflow blocks
- Fix dependency setting, ensure public fields when interpreter is generated into a separate package
- Better validation errors when parsing openapi
- Add comparator helpers
- Add cloner helpers
- Generate constructor for object with default values
- Refactor block generation, allow to generate separate files for methods
- Add AnyOf and OneOf to json schema, validation fixes
- Remove AnyOf, as it won't be supported
- Change type checking to io.ReadCloser in bytestream
